Source/SelfService/Web/api/api.ts

```diff
@@ -156,9 +156,9 @@ export async function deleteMicroservice(applicationId: string, environment: str
     return result.status === 200;
 }
 
-export async function saveMicroservice(input: any): Promise<boolean> {
+export async function saveMicroservice(input: any): Promise<Response> {
```

Comment:
I think it would make a little sense do to more of the work here in checking that everything went okay.

What you could do is `await` the fetch call to get a response, then check that response if the `status code == 200`, and if not throw an error (could try to parse the body as json or just take the body text as the error).

If it was 200, then return the result after calling `.json()` on it, that would make this method return a nice result (seems like it is som actual good content there) or throw an error. What do you think?

Reply:
Changed it to throw an exception with the response message as the exception message